http-server/src/main/java/io/micronaut/http/server/exceptions/UnsatisfiedRouteHandler.java

```diff
@@ -55,7 +55,7 @@ public UnsatisfiedRouteHandler(ErrorResponseProcessor<?> responseProcessor) {
     @Override
     public HttpResponse handle(HttpRequest request, UnsatisfiedRouteException exception) {
         if (LOG.isTraceEnabled()) {
-            LOG.trace("{} (Bad Request): {}", request, exception.getMessage());
+            LOG.trace("{} (Bad Request): {}", request, exception.getMessage(), exception);
```

Comment:
For `trace` level it might make sense to keep this added exception stacktrace, but I'll leave that up to the review. Pointing it out such that it's not missed.

Reply:
```suggestion
            LOG.trace("{} (Bad Request): {}", request, exception.getMessage());
```